plugins/postcss-nesting/src/lib/atrule-within-atrule.ts

```diff
@@ -1,16 +1,12 @@
+import type { AtRule } from 'postcss';
 import cleanupParent from './cleanup-parent.js';
 import mergeParams from './merge-params.js';
 import shiftNodesBeforeParent from './shift-nodes-before-parent.js';
 import validAtrules from './valid-atrules.js';
 
-/*
- * DEPRECATED: In v7.0.0 these features will be removed as they are not part of
- * the nesting proposal.
- */
```

Comment:
I think it might have become a part of the spec after this comment was added : https://www.w3.org/TR/css-nesting-1/#conditionals